libs/interpret/src/lib/MLIDashboard/Controls/MultiICEPlot/MultiICEPlot.tsx

```diff
@@ -549,7 +549,7 @@ export class MultiICEPlot extends React.PureComponent<
 
   private buildRangeView(featureKey: string): IRangeView | undefined {
     const summary = this.props.jointDataset.metaDict[featureKey];
```

Comment:
I thought there is another issue where we are not passing the categorical mapping for features from python side to UI side?